### ISSUE TITLE
Design for color blindness by replacing red/green with orange/blue

### ISSUE THREAD

**Issue by nya-elimu** [elimu-ai/webapp#1458]
# Storybook contribution events
- URL: `/content/storybook/edit/<ID>`
- E.g. http://hin.elimu.ai/content/storybook/edit/17#contribution-event_267
  <img width="386" alt="Screen Shot 2021-10-15 at 2 36 59 PM" src="https://user-images.githubusercontent.com/15718174/137474530-249a23aa-ed4a-4917-b99e-647cb2a93a4f.png">

# Storybook peer-review events
- URL: `/content/storybook/edit/<ID>`
- E.g. http://hin.elimu.ai/content/storybook/edit/17#contribution-event_267
  <img width="550" alt="Screen Shot 2021-10-19 at 2 08 21 PM" src="https://user-images.githubusercontent.com/15718174/137889405-58a57703-f760-4744-b43d-61c242567b0d.png">
  <img width="541" alt="Screen Shot 2021-10-19 at 2 08 31 PM" src="https://user-images.githubusercontent.com/15718174/137889411-ae9f9f58-de88-457c-913f-972fd5e12f57.png">

# Contributor overview
- URL: `/content/contributor/<ID>`
- E.g. http://hin.elimu.ai/content/contributor/41
  <img width="636" alt="Screen Shot 2021-10-19 at 2 10 52 PM" src="https://user-images.githubusercontent.com/15718174/137889844-0522b072-59a6-4251-8cd5-1d95ca2e3ee9.png">
  <img width="917" alt="Screen Shot 2021-10-19 at 2 10 42 PM" src="https://user-images.githubusercontent.com/15718174/137889868-daefae20-bd2f-4670-9cd4-2cbc75b60f86.png">

**Comment by codingmickey** [comment]
Hey wanted to know more about this issue, like where does this screen come and can you give more info about it

**Comment by nya-elimuai** [comment]
> Ok thank you now it's easy to understand, so can I use any shades of orange and blue? or any specific shades?

@codingmickey Yes, feel free to pick any shades you think will look good. Here is a color palette you might find useful: https://materializecss.com/color.html

**Comment by codingmickey** [comment]
Just curious is this project being continued? If yes, I'd like to help in any way possible 😉

**Comment by nya-elimu** [comment]
@codingmickey Yes, it is being continued, and your help is highly appreciated 🙂

We're focusing on making it work with Hindi and Tagalog  languages. You are welcome to join the community chat also: https://github.com/elimu-ai/wiki#community-chat-
